src/main.py

```diff
@@ -9,6 +9,6 @@
 
 app.register_blueprint(blueprint)
 
-# Dummy comment
 if __name__ == "__main__":
-    app.run(debug=True, host="0.0.0.0")
+    debug = True if Config.ENVIRONMENT == "dev" else False
```

Comment:
Instead of comparing the value of ENVIRONMENT, you can define the env var DEBUG as `DEBUG=True` and use the value here: `app.run(debug=Config.DEBUG, host="0.0.0.0")`. You save one if statement.